src/main/java/com/example/demo/handler/MedicalClinicExceptionHandler.java

```diff
@@ -24,4 +25,16 @@ public ErrorMessage handlePatientNotFoundException(PatientNotFoundException ex)
     public ErrorMessage handlePatientAlreadyExistException(PatientAlreadyExistException ex) {
         return new ErrorMessage(ex.getMessage(), HttpStatus.BAD_REQUEST, new Date());
     }
+
+    @ExceptionHandler(CannotChangeIdCardNoException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    public ErrorMessage handleCannotChangeIdCardNoException(CannotChangeIdCardNoException ex) {
+        return new ErrorMessage(ex.getMessage(), HttpStatus.BAD_REQUEST, new Date());
+    }
+
+    @ExceptionHandler(NullPointerException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    public ErrorMessage handleNullPointerException(NullPointerException ex) {
+        return new ErrorMessage("Data cannot be null", HttpStatus.BAD_REQUEST, new Date());
```

Comment:
To do zmiany ze względu na to, że wyjątek NPE może polecieć zawsze. Co moze wprowadzać w błąd